File: src/main/java/org/zenbot/szolnok/timetable/domain/Schedule.java
package org.zenbot.szolnok.timetable.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Data
@Document
@NoArgsConstructor
@AllArgsConstructor
public class Schedule {

    @Id
    private String id;
    private List<BusArrival> busArrivals;
}

File: src/main/kotlin/org/zenbot/szolnok/timetable/domain/Schedule.kt
package org.zenbot.szolnok.timetable.domain

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class Schedule(
    @Id var id: String = "",
    var busArrivals: List<BusArrival> = ArrayList()
)
